src/sentry/tasks/update_user_reports.py

```diff
@@ -16,7 +19,11 @@ def update_user_reports(**kwargs):
         group__isnull=True, environment__isnull=True, date_added__gte=now - timedelta(days=1)
     )
 
+    count = 0
     for report in user_reports:
         event = eventstore.get_event_by_id(report.project_id, report.event_id)
         if event:
             report.update(group_id=event.group_id, environment=event.get_environment())
+            count += 1
+
+    logger.info("update_user_reports.records_updated", extra={"records_updated": count})
```

Comment:
Do you also want to log how many candidates were supposed to be updated?

Reply:
Good call